test/views/RepresentativeNameViewSpec.scala

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import base.ViewBaseSpec
+import forms.RepresentativeNameFormProvider
+import messages.{BaseMessages, RepresentativeNameMessages}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.data.Form
+import play.api.mvc.Call
+import play.twirl.api.Html
+import views.html.RepresentativeNameView
+
+class RepresentativeNameViewSpec extends ViewBaseSpec with BaseMessages {
+
+  private lazy val injectedView: RepresentativeNameView = app.injector.instanceOf[RepresentativeNameView]
+
+  val formProvider: RepresentativeNameFormProvider = injector.instanceOf[RepresentativeNameFormProvider]
+
+  "Rendering the Importer's Name page" when {
+    "no errors exist" should {
+
+      val form: Form[String] = formProvider.apply()
+      lazy val view: Html = injectedView(form, Call("GET", controllers.routes.UserTypeController.onLoad().url))(fakeRequest, messages)
```

Comment:
controllers.routes.UserTypeController.onLoad() is of type Call so you do not need to create a new call object.